frontend/src/hooks/useLanguage.js

```
@@ -159,6 +255,97 @@ const translations = {
+    notifications: 'Notifications', noNotifications: 'No notifications',
+    markAllRead: 'Mark all read', overdue: 'Overdue', dueToday: 'Due today', dueTomorrow: 'Due tomorrow',
+    completed: 'Completed', pending: 'Pending', all: 'All', search: 'Search',
+    addFirstTask: 'Add your first task!', focusMode: 'Focus Mode', startFocus: 'Start Focus',
+    pause: 'Pause', resume: 'Resume', reset: 'Reset', ready: 'Ready', focusing: 'Focusing', paused: 'Paused',
+    sessionsToday: 'Sessions today', generatingPlan: 'Generating plan...', noPendingTasks: 'No pending!',
+    addTasksFromDashboard: 'Add from dashboard', loadingCalendar: 'Loading...',
+    refresh: 'Refresh', tasksPending: 'Pending', completedToday: 'Done today',
+    recommendedTaskOrder: 'Recommended', noTasks: 'No tasks', askMeAnything: 'Ask me...',
+    send: 'Send', noLanguagesFound: 'No found', enterYourEmail: 'Your email',
+    enterPassword: 'Password', defaultDuration: '60',
+  },
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)